ironmon_tracker/Tracker.lua

```diff
@@ -410,7 +410,9 @@ end
 
 -- If the viewed Pokemon has the move "Hidden Power" (id=237), return it's tracked type; otherwise default type value = NORMAL
 function Tracker.getHiddenPowerType()
-	local viewedPokemon = Battle.getViewedPokemon(true)
+	-- Default to 'true' if not using open book setting
+	local useOpenBookInfo = not Options["Open Book Play Mode"] or Tracker.Data.isViewingOwn
+	local viewedPokemon = Battle.getViewedPokemon(useOpenBookInfo)
```

Comment:
I'm a bit confused by the variable naming here of `useOpenBookInfo`, it's set to true if the option for open book is disabled / we are *not* using open book?

Also a bit confused as to what this bit of code is doing overall here as opposed to what the removed line was doing, in regards to the desired behaviour both with / without open book setting enabled

Reply:
The labels here are rough, but ultimately its a change from only ever showing hidden power type for *your* pokemon, and allowing it to show the hidden power type of the enemy pokemon.

Which I only just now realized doesnt actually show the hidden power type as defined by the game, and just shows the type you've tracked. Whoops. I'll revert this change.

Reply:
Yeah the latter is why I was confused about the whole usage, though even if it did work like that i'd still feel unsure about showing hidden power types given that it's derived from IVs (and not info from the log), as mentioned in the other comment